src/algebra/add_torsor.lean

```diff
@@ -49,8 +49,7 @@ class has_vsub (G : out_param Type*) (P : Type*) :=
 infix ` +ᵥ `:65 := has_vadd.vadd
 infix ` -ᵥ `:65 := has_vsub.vsub
 
-section prio
-set_option default_priority 100 -- see Note [default priority]
```

Comment:
Maybe this PR should remove the "default priority" library note as well.

Reply:
This PR only removes about half of these `default_priority` things, but the next PR will do so!

Reply:
Ok!